Tests/UnitTests/Networking/HTTPClientTests.swift

```diff
@@ -637,6 +637,37 @@ class HTTPClientTests: XCTestCase {
         expect(completionCalled.value).toEventually(equal(true), timeout: .seconds(1))
     }
 
+    func testGetsResponseFromETagManagerWhenStatusCodeIsNotModified() {
```

Comment:
Turns out there was 0% coverage for `HTTPClient` caching responses with `ETagManager` 😬